Assets/Prefabs/Bridge.prefab

```diff
@@ -133,9 +149,9 @@ AudioSource:
   m_Enabled: 1
   serializedVersion: 4
   OutputAudioMixerGroup: {fileID: 0}
-  m_audioClip: {fileID: 8300000, guid: 1ca0270ba072e48498cbebefe84187ce, type: 3}
-  m_PlayOnAwake: 0
-  m_Volume: 0.2
+  m_audioClip: {fileID: 0}
+  m_PlayOnAwake: 1
```

Comment:
Should it play on awake?

Reply:
I don't actually know if the sound was implemented, if it should play sound on spawn, yes, otherwise no.

Reply:
Stick with previous

Reply:
Our other scripts have a reference to the audio that they manually call .play on so don't need PlayOnAwake